core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -1640,6 +1640,18 @@ class LogTest {
     assertEquals(partition.toInt, topicPartition.partition)
   }
 
+  // see https://issues.apache.org/jira/browse/KAFKA-5232
```

Comment:
This is simple enough that we can write a short description here instead of referencing the JIRA.